src/components/Pages/Unit/Container.js

```diff
@@ -46,7 +46,8 @@ export class UnitContainer extends Component {
 
     const { id } = match.params;
     if (
-      unit
+      componentIsMounted
```

Comment:
It means that data is never requested on componentDidMount?  So maybe just not to request it there?
BTW, is it ok?